features/step_definitions/capcha_steps.rb

```diff
@@ -7,7 +7,7 @@
 end
 
 Then /^I should see the captcha$/ do
-  page.should have_selector(RECAPTCHA_SCRIPT)
+  page.should have_selector(RECAPTCHA_SCRIPT, visible: false)
```

Comment:
I asked Josemi about this, because I find very weird the `I should see the captcha` in the title and then in this line `visible: false`, and he told me that it is because it is a `<script>…</script>` so it is not visible in the browser.

Keeping in mind that he didn't add this, just refactored, ok 👍 Just adding the comment here because the reason is not obvious 😄